index.html

```diff
@@ -1,9 +1,77 @@
-Experience
-BeachMart.Inc, Harris Teeter, Junior Project, Asachi Lyceum. 
-Connect
-https://github.com/DoinaFitchevici
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <main>
+    <section>
+    <h1>About</h1>
+    <p>
+    Hello and welcome to my portfolio website! My name is Doina Fitchevici, I am an aspiring front-end web developer, and also a mother of a two-year old boy. Besides enjoying being a mother, I am thrilled to embark on this journey to pursue my passion for web development and share my journey with you.
+    Although I may currently lack professional experience in the field, I have dedicated significant time and effort to learning and honing my skills as a front-end web developer.
+    </p>
+    </section>
+    <section>
+    <h1>Experience</h1>
+    <h4>Stay At Home Mom</h4>
+    <h5>Self-Employed</h5>
+    <p>2021 - present</p>
+    <p>
+    <ul>
+    <li>Spent two years parenting a baby boy.</li>
+    <li>Networked on social media with other moms and maintained a Facebook group that organized kids events and play-dates, swapped clothes and exchanged parenting advice.</li>
+    <li>Started a network marketing business with Amway, made customers and built a team.</li>  
+    <li>Completed an intro to programming course at Code the Dream and honed these skills:</li>
+    <ul>
+    <li>Proficiency in programming languages like HTML, CSS, and JavaScript</li>
+    <li>Proficiency in using Git version control system to track changes, collaborate with others, and efficiently manage code repositories</li>
+    <li>Debugging and problem-solving</li>
+    <li>Familiarity with web development tools and software, such as Visual Studio Code text editor</li>
+    <li>Committment to continuous learning and keeping up with the latest trends, technologies, and frameworks in the ever-evolving field of web development</li>
+    </ul>
+    </ul>
+    </p>
+    <h4>Retail Manager</h4>
+    <h5>BeachMart.Inc</h5>
+    <p>2016 - 2021</p>
+    <p>
+    <ul>
+    <li>Provided superior customer service and met standards set out by the company.</li>
+    <li>Ensured merchandise presentation, cleanliness, and safety.</li>
+    <li>Received merchandise, priced, and dislayed in the stores.</li>
+    </ul>
+    </p>
+    <h4>Child Care Teacher</h4>
+    <h5>Junior Project</h5>
+    <p>2014-2015</p>
+    <p>
+    <ul>
+    <li>Created and maintained a full schedule of activities and teaching materials.</li>
+    <li>Interacted with parents and updated them on the progress of their children.</li>
+    <li>Ensured compliance with teaching standards and safety regulations. </li>
+    </ul>
+    </p>
+    </section>
+    <section>
+    <h1>Connect</h1>
```

Comment:
I would reserve `h1` for the page title, with sections inside the page being `h2` or similar.